packages/hint-doctype/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     ],
     "references": [
         { "path": "../hint" },
+        { "path": "../parser-html" },
```

Comment:
I'm going to update the references en master and push a change directly so we don't have this files modified in here